__tests__/pages/admin/lessons/new/lessons.test.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react'
+import { gql } from '@apollo/client'
+import Lessons from '../../../../../pages/admin/lessons/new/index'
+import { MockedProvider } from '@apollo/client/testing'
+import { act, render, waitFor } from '@testing-library/react'
+import dummyLessonData from '../../../../../__dummy__/lessonData'
+import dummySessionData from '../../../../../__dummy__/sessionData'
+import dummyAlertData from '../../../../../__dummy__/alertData'
+import '@testing-library/jest-dom'
+import GET_APP from '../../../../../graphql/queries/getApp'
+
+const EXERCISES = gql`
```

Comment:
This should be in the queries folder(look for my other comment about this, I provide link in it)